code/modules/tgui/modal.dm

```diff
@@ -0,0 +1,370 @@
+/**
+ * tgui modals
+ *
+ * Allows creation of modals within tgui.
+ */
+
+GLOBAL_LIST(tgui_modals)
```

Comment:
I feel like this should be tracked as a variable on the subsystem itself

Reply:
I disagree, this extension is designed to be self-contained (almost at least, the defines are in `__DEFINES`) to make modifying it or adding it in to another codebase easier. Adding more to the list of files it touches would ruin the point of it